docs/incident_attributes.rst

```diff
@@ -292,10 +292,10 @@ The :class:`Event <Event>` object in this scenario will have the following attri
 
 Port Scans
 ==========
-There are five types of port scans incidents.
+There are five types of port scan incidents.
 
-#. A host port scan occurs when a single Canary is port scanned by a single source.
-#. A consolidated network port scan occurs when multiple Canaries are scanned by a single source.
+#. A host port scan occurs when a single Canary is a port scanned by a single source.
```

Comment:
This change ("a") isn't right, we should undo the change.

Reply:
updated 🔨